src/components/IGVFileSelector.js

```diff
@@ -34,7 +34,7 @@ const isTdrUrl = (fileUrl) => {
 };
 
 const findIndexForFile = (fileUrl, fileUrls) => {
-  if (!['.cram', '.bam', '.vcf'].some((extension) => fileUrl.endsWith(extension))) {
+  if (!['.cram', '.bam', '.vcf', '.gz'].some((extension) => fileUrl.endsWith(extension))) {
```

Comment:
I am not an IGV expert but `.gz` seems too permissive to me, as I think the only gzip files that are supported are vcfs. Could this use `.vcf.gz` instead?

See https://igv.org/doc/desktop/#FileFormats/DataTracks/#vcf

Reply:
It looks like `gz` is mapped to `.gz.tbi` below? But that is a good question, what would happen if a user tries to load a different type of compressed file that does not untar to .tbi?

Reply:
I agree with you but it's annoying with how the code is structured right now.

`getValidIgvFiles` uses `splitExtension` which considers only the last segment `.gz` to be the extension.

Meanwhile, we have a duplicative allowlist in `findIndexForFile` that applies `endsWith` and matches `.vcf.gz`.

I'll poke at it some more and see if I can improve things.

Reply:
> what would happen if a user tries to load a different type of compressed file that does not untar to .tbi

Nothing bad, the line `gz: [`${base}.gz.tbi`]` would not match anything and we would not enable IGV.

Reply:
Gotcha, in that case I am not supper worried about adding `gz` as is honestly

Reply:
I also don't see a practical issue with adding `.gz` as is, but only adding support for `.vcf.gz` as Phil suggests would in principle suffice for this use case.  BAM and CRAM essentially never appear with `.gz` suffixes.  BAM is gzipped (via bgzip) by definition, making the `.gz` suffix superfluous.

IGV supports other genomics data files that appear with and without a .gz suffix.  [BED files](https://genome.ucsc.edu/FAQ/FAQformat.html) are another common genomics data format that often appears as `${base}.gz`, e.g. `foo.bed.gz`.  These often pair with a TBI file, e.g. `foo.bed.gz.tbi`.  If you open Chrome DevTools, go to Network panel, then go to the public study [SCP2699 in Single Cell Portal](https://singlecell.broadinstitute.org/single_cell/study/SCP2699/multiome-demo-study-10x-cell-ranger-arc-processed?genes=PTEN&tab=genome#study-visualize), you can see an example `.bed.gz` and `.bed.gz.tbi` pair being fetched to render the igv.js track with grey and red features.

[Others genomic file formats](https://genome.ucsc.edu/FAQ/FAQformat.html) viewable in IGV that are slightly less common but foreseeable include GFF / GFF3 and GTF with and without `.gz` suffixes, e.g. `foo.gff`, `foo.gff.gz`, `foo.gff3.gz`, `foo.gtf`, `foo.gtf.gz`.  

Those other file extensions aren't necessarily in scope here, but hopefully the extra context helps.